tests/config_validator/test_validate.py

```diff
@@ -1,109 +1,114 @@
-# pylint: disable=missing-function-docstring,missing-module-docstring
+# pylint: disable=missing-function-docstring,missing-module-docstring,bad-continuation
```

Comment:
We now have a [`.pylintrc` for tests](https://github.com/ChauffeurPrive/nestor-api/blob/master/tests/.pylintrc#L144) that already handle those rules, except `bad-continuation` that should be avoided as much as possible. But, if necessary, I think that we are use to disabling it only for the problematic line

Reply:
I didn't added this to each line because it would have been on each line of the multiline method definition, not nice
But I removed unused pylintinstruction in the rest  of the project while  at it https://github.com/ChauffeurPrive/nestor-api/pull/40/commits/4aea78e3bba6a52c35e1e780c88a2250c15284fc